### PR TITLE
Updated gradle, wrapper, xp version and dependencies.

### PR DESCRIPTION
**Pull request by tinaeile**

### CODE REVIEW

**Comment by Roeland-Bouvet** [comment]
Hmmm, I tried to get the jar on jitpack, but it seems like there might be some things missing. See log here https://jitpack.io/com/github/bouvet-apps/app-favicon/abedcaae16/build.log

Have you had a look at this readme https://github.com/bouvet/team-enonic-commons/blob/main/docs/en/publish-to-enonic-market.md ? :)

**Comment by tinaeile** [comment]
> Hmmm, I tried to get the jar on jitpack, but it seems like there might be some things missing. See log here https://jitpack.io/com/github/bouvet-apps/app-favicon/abedcaae16/build.log
> 
> Have you had a look at this readme https://github.com/bouvet/team-enonic-commons/blob/main/docs/en/publish-to-enonic-market.md ? :)

Thanks, I'll take a look at it! 😊

File: gradle.properties [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-xpVersion = 7.0.0
 version = 2.0.1
+
+# XP App values
+appDisplayName = Favicon
+appName = io.selbekk.favicon
+xpVersion = 7.9.0
+
+# Settings for publishing to a Maven repo
+group = io.selbekk
```
Comment (SkylineTardis) [inline gradle.properties]: 🙈 newline?

File: build.gradle [hunk 2]
```diff
@@ -27,6 +27,6 @@ repositories {
 }
 
 wrapper {
-  gradleVersion = '5.6'
+  gradleVersion = '8.3'
   distributionType = Wrapper.DistributionType.BIN
 }
```
Comment (SkylineTardis) [inline build.gradle]: 🙈

**Review by SkylineTardis — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (tinaeile) [reply gradle.properties]: Done 🥳

File: build.gradle (hunk 2 above)
Comment (tinaeile) [reply build.gradle]: Done 🥳

**Comment by tinaeile** [comment]
@Roeland-Bouvet Looks like its working now: https://jitpack.io/com/github/bouvet-apps/app-favicon/429fd1f2db/build.log 🚀

**Review by Roeland-Bouvet — APPROVED** [review]

**Review by andersnin — APPROVED** [review]

**Pull request merged**